Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -1719,6 +1723,7 @@ class RenderStyle {
     static OptionSet<Containment> strictContainment() { return OptionSet<Containment> { Containment::Size, Containment::Layout, Containment::Paint, Containment::Style }; }
     static OptionSet<Containment> contentContainment() { return OptionSet<Containment> { Containment::Layout, Containment::Paint, Containment::Style }; }
     static ContainerType initialContainerType() { return ContainerType::Normal; }
+    static ContentVisibility initialContentVisibility() { return ContentVisibility::Visible; }
```

Comment:
In new code I think we should use `constexpr` as well as `static` for functions like this.

Reply:
Done.